content/guides/understanding-inbox-messaging.md

```diff
@@ -0,0 +1,77 @@
+# Understanding Inbox Messaging
+
+---
+
+1. TOC
+{:toc}
+
+---
+
+## Preface
+
+BookingSync provides the ability to integrate with the Inbox on multiple different channels/platforms we support. This guide will present
+how to implement this integration.
+
+
+## Purpose
+
+The purpose of this feature is to provide a seamless integration between internal messaging systems in certain Channels and the BookingSync platform, for adminstrators to be able to handle incoming inquiries in a centralized location.
```

Comment:
@Kefa7y I would avoid mentioning `inquiries` in this context. Inquiry is an actual model and endpoint in our API so it might create confusion. For now it is just best to mention property manager <-> guest communication.